packages/api/src/graphql/query.private.ts

```diff
@@ -435,7 +435,35 @@ export const GraphQLQuery = new GraphQLObjectType<undefined, Context>({
           hasMore = listResult.pageInfo.hasNextPage
           afterCursor = listResult.pageInfo.endCursor
         }
-        return mapSubscriptionsAsCsv(subscriptions)
+
+        // fixes https://wepublish.atlassian.net/browse/WPC-928
+        const subscriptionsWithUser: Subscription[] = []
+        const users: User[] = []
+        hasMore = true
+        afterCursor = undefined
+        while (hasMore) {
+          const userResults: ConnectionResult<User> = await dbAdapter.user.getUsers({
+            limit: Limit(100),
+            sort: UserSort.CreatedAt,
+            cursor: InputCursor(afterCursor ?? undefined),
+            order: SortOrder.Descending
+          })
+          users.push(...userResults.nodes)
+          hasMore = userResults.pageInfo.hasNextPage
+          afterCursor = userResults.pageInfo.endCursor
+        }
+
+        // map users on subscriptions
+        for (const subscription of subscriptions) {
+          const userIndex = users.findIndex(tmpUser => tmpUser.id === subscription.userID)
+          if (userIndex < 0) {
+            subscriptionsWithUser.push(subscription)
```

Comment:
SInce you not always got an user (if userIndex = -1) the name subscriptionsWithUser is not very descriptive. Maybe better something like subscriptionUserMap or something better. :-)